Bottom: 4773f6420ee8101dd2e6c78e404cd88bdef59229
Top:    e6c3414b507a56c0c338d78a5d3778ca60c149e9
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-01-25 09:52:46 +0800

Refresh of debug

---

diff --git a/drivers/vhost/vhost.c b/drivers/vhost/vhost.c
index 4425d10..e1e91d3 100644
--- a/drivers/vhost/vhost.c
+++ b/drivers/vhost/vhost.c
@@ -2035,6 +2035,8 @@ static int vhost_read_indices(struct vhost_virtqueue *vq, u16 num)
 		last_avail_idx = (last_avail_idx + 1) & (vq->num - 1);
 	}
 
+	vq->last_avail_idx += ret;
+
 	/* Only get avail ring entries after they have been exposed by guest. */
 	smp_rmb();
 	indices->head = ret;
